tests/integration_tests/multi_source/main.go

```diff
@@ -194,8 +194,8 @@ func truncateDDL(ctx context.Context, db *sql.DB) {
 
 func ignoreableError(err error) bool {
 	knownErrorList := []string{
-		"Error 1146:", // table doesn't exist
-		"Error 1049",  // database doesn't exist
+		"Error 1146", // table doesn't exist
```

Comment:
Why change the test?

Reply:
After we upgrade go-sql-driver, the error message is changed, for example 

Error 1142 **(42000)**: ALTER command denied